src/natcap/invest/seasonal_water_yield/seasonal_water_yield.py

```diff
@@ -659,33 +659,15 @@ def execute(args):
     output_align_list = [
         file_registry['lulc_aligned_path'], file_registry['dem_aligned_path']]
     if not args['user_defined_local_recharge']:
-        precip_path_list = []
-        et0_path_list = []
-
         et0_dir_list = [
             os.path.join(args['et0_dir'], f) for f in os.listdir(
                 args['et0_dir'])]
         precip_dir_list = [
```

Comment:
Do you think it makes sense to do these list comprehensions within the `_get_monthly_file_lists` function? It seems like logically they are closely related to the rest of the code in the function, so it might be nice to encapsulate it there. And then the function would take `args['et0_dir']` as a direct input.

Reply:
Yep good call!